include/stout/flags/flags.cc

```diff
@@ -663,6 +670,52 @@ void Parser::Parse(const std::vector<ArgumentInfo>& values) {
 
 ////////////////////////////////////////////////////////////////////////
 
+void Parser::SetFieldMessageOrAggregateErrors(
+    const std::string& value,
+    const std::string& name,
+    const google::protobuf::FieldDescriptor* field,
+    google::protobuf::Message* message,
+    std::set<std::string>& errors) {
+  // Parse the value using an error collector that aggregates the
+  // error for us to print out later.
+  struct ErrorCollector : public google::protobuf::io::ErrorCollector {
+    void AddError(
+        int /* line */,
+        int /* column */,
+        const std::string& message) override {
+      error += message;
```

Comment:
Can/should we include the line and column number for easier debugging?

Reply:
Looks like we're still not including line / column information in the error that we add: should we add a TODO to do so if that's not something that we can figure out right now?